utils/src/check/super.c

```diff
@@ -241,8 +241,18 @@ int check_supers(int data_fd)
 	debug("super magic 0x%04x", global_super->hdr.magic);
 
 	debug("Superblock flag: %llu", global_super->flags);
-	if (global_super->flags != SCOUTFS_FLAG_IS_META_BDEV)
+	if (global_super->flags != SCOUTFS_FLAG_IS_META_BDEV) {
 		problem(PB_SB_BAD_FLAG, "Bad flag: %llu expecting: 1 or 0", global_super->flags);
+		if (repair) {
+			global_super->flags &= SCOUTFS_FLAG_IS_META_BDEV;
```

Comment:
`global_super` is always set for the meta device. So I would just assign the flag, instead of modifying the incorrect value. The repaired value must always be equal to `SCOUTFS_FLAG_IS_META_BDEV`.